lib/btrfs_test.pm

```diff
@@ -72,6 +74,17 @@ sub snapper_nodbus_restore {
     die 'DBus service should be inactive, but it is active' if ($ret == 0);
     script_run('systemctl default', timeout => 600, die_on_timeout => 0);
     my $tty = get_root_console_tty;
+
+    if (is_sle('<15-SP3') && !defined(my $match = check_screen("tty$tty-selected", 120))) {
+        record_soft_failure("bsc#1185098 - logind fails after return back from rescue");
```

Comment:
Also, Oli had a valid point about WON'T FIX bugs, this seems to be one of them. By doing soft-failure, our jobs will be always soft-failed and will keep posting comments to the ticket, I wonder if we should just use record_info...

Reply:
That is a valid point, we can change that when there will be some response from the upstream. As of now, there are no comments, so maybe they will consider fixing it.

Reply:
Makes sense.